src/site/content/en/lighthouse-best-practices/errors-in-console/index.md

```diff
@@ -2,82 +2,75 @@
 layout: post
 title: Browser errors were logged to the console
 description: |
-  Learn about `errors-in-console` audit.
+  Learn how to identify and fix browser errors.
 web_lighthouse:
   - errors-in-console
+updated: 2019-08-28
 ---
 
-Lighthouse flags any browser errors logged to the console:
+Most browsers ship with built-in developer tools.
+These developer tools usually include a [console](https://developers.google.com/web/tools/chrome-devtools/console/).
+The console gives you information about the page that's currently running.
+
+Messages logged in the console have a severity level:
+either `Verbose`, `Info`, `Warning`, or `Error`.
+An `Error` message means there's a problem on your page that you need to resolve.
+
+## How the Lighthouse browser error audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags all browser errors logged to the console:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="errors-in-console.png" alt="Lighthouse audit showing browser errors in the console">
-  <figcaption class="w-figcaption">
-    Figure 1. Browser errors logged to the console.
-  </figcaption>
+  <img class="w-screenshot" src="errors-in-console.png" alt="Lighthouse audit showing browser errors in the console">
 </figure>
 
-## How this audit fails
+{% include 'content/lighthouse-best-practices/scoring.njk' %}
 
-Most browsers ship with built-in developer tools.
-These developer tools usually include a console.
-The console gives you information about a page while that page is running.
-Ultimately,
-the messages you see in the console either come from the web developers who built the page,
-or the browser itself.
-
-When someone logs a message to the console,
-they can indicate the importance, or "severity level" of the message.
-An "error" message is an important message
-representing an unresolved failure in the page.
-In other words,
-when you see an error, the page isn't running as intended.
+## How to fix browser errors
+
+Fix each browser error that Lighthouse reports
+to ensure that your page runs as expected for all your users.
+
+Chrome DevTools includes a couple tools
+to help you track down the cause of errors:
+
+- Below the text of each error, the console shows the
```

Comment:
I always capitalize `Console` in the context of Chrome DevTools. It's a proper noun.

Reply:
So, bit of strangeness here: earlier in the post we talk about consoles in the abstract, so the word isn't capitalized. I think it'll seem like an error if we start capitalizing midway through. To better signal this is a proper noun, would it make sense to go with "DevTools Console" or something?

Reply:
Yes this post is awkward so please do whatever makes the most sense to you. I'm more just noting as general FYI that we should capitalize `Console` when talking about the proper noun in DevTools.

Reply:
Another option is `the Console panel` which is a common phrasing throughout the DevTools docs